doc/content/devices/models/sensative-strips/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Sensative Strips"
+description: ""
+weight: 
+---
+
+{{< figure src="strip.png" alt="Generic Node Sensor Edition" class="float plain" width="80%">}}
+
+The Sensative [Strips](https://sensative.com/sensors/strips-sensors-for-lorawan/) is a multi-sensor family with multi-purposes, like humidity, temperature, light (LUX) measuring, and magnetic open/close sensor (suitable for windows or door monitoring).
+
+<!--more-->
+
+## Provisioning Information
+
+Provisioning information (The **JoinEUI/AppEUI**, **DevEUI** and **AppKey**) can be found on a sticker inside the box. 
+
+{{< figure src="strips-prov-info.png" alt="Strips provisioning information" width="50%">}}
+
+{{< note "The Strips does not support onboarding with a QR code. You can onboard it in the section below." />}}
+
+## Onboarding to {{% tts %}}
+
+The device can be onboarded using the Device Repository or by manually entering the specific information.
+The [LoRaWAN® Device repository](https://github.com/TheThingsNetwork/lorawan-devices) comprises over 600 end-device profiles, including the Sensative Strips, which enables quick and easy registration with {{% tts %}}.
+
+In the **End device type** section, under **Input method**, select either the **Select the end device in the LoRaWAN Device Repository** or **Enter end device specifics manually** option. The [Onboarding devices without QR codes]({{< ref "/devices/adding-devices/#onboarding-devices-without-qr-codes" >}}) section of the [Adding Devices]({{< ref "/devices/adding-devices/" >}}) guide explains this procedure in detail. Following details are Strips-specific:
+
+- End device brand: **Sensative AB**
+- Model: **Strips**
+- Hardware Ver.: **1.0**
+- Firmware Ver.: **1.0**
+
+{{< note >}} If you are manually adding the device, enter these details specifically for the Strips:
+- LoRaWAN version: **LoRaWAN Specification 1.0.3**
+- Regional parameters version: **RP001 Regional Parameters 1.0.3 revision A** {{</ note >}}
```

Comment:
Same comment for this part as in https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/1240